src/pulseaudio_dbus/init.lua

```diff
@@ -349,6 +349,24 @@ function pulse.Device:toggle_muted()
   return self:is_muted()
 end
 
+--- Get the current active port object path
+-- @return the active port object path
+-- @return nil if no active port is set
+-- @see pulse.Device:set_active_port
+function pulse.Device:get_active_port()
+  return self:Get("org.PulseAudio.Device", "ActivePort")
```

Comment:
I'm not sure about accepting/returning objects. Getters and properties returning the object path as a string was a little odd when learning the API but ends up being pretty consistent, not only with the rest of pulseadio_dbus but with the pulseaudio docs, etc.
I'm not against it though. Maybe it's a good idea to avoid boilerplate code. But I think it should be done in a future PR, addressing it for all getters/setters in which makes sense. i.g. get/set_fallback_source/sink.

Reply:
Yeah, I agree. It might be a nice feature for the future.